### DISCUSSION TITLE
Wallet with PlotNFT - how to delete unused wallets?

### DISCUSSION THREAD

**Discussion by dominiaz** [Chia-Network/chia-blockchain#7291] [General]
I used 3x command `chia plotnft create -u <URL> -s pool` with the URL of the same pool.

So chia created me 3 extra wallets for pooling on the same pool.

I can`t find any option to remove 2 unused wallets.

How to rremove those wallets?

**Comment by mariano54** [comment]
You cannot currently delete pooling wallets (plot NFTs)
